tests/riscof/Makefile

```diff
@@ -31,7 +31,7 @@ arch%: config$$*.ini | $(work_dir) $(arch_workdir)
 # Generate wally-riscv-arch-test targets
 wally%: config$$*.ini | $(work_dir) $(wally_workdir)
 	riscof run --work-dir=$(work_dir)/$@ --config=$< --suite=$(wally_dir)/riscv-test-suite/ --env=$(wally_dir)/riscv-test-suite/env --no-browser --no-dut-run
-	$(MAKE) -f makefile-memfile WORKDIR=$(work_dir)/$@
+	$(MAKE) -f makefile-memfile --jobs WORKDIR=$(work_dir)/$@
```

Comment:
Ideally make is called using the `--jobs` or `--jobs #` flag which will then let it run all sub makes using that job server. Add `--jobs` inside the Makefile can cause some weird issues with resetting the job count that it uses.